packit/patches.py

```diff
@@ -231,6 +235,76 @@ def linearize_history(git_ref: str):
             env={"FILTER_BRANCH_SQUELCH_WARNING": "1"},
         )
 
+    @staticmethod
+    def process_patches(
+        patches: Dict[str, str], commits: List[git.Commit], patch_list: List
+    ):
+        """ process patches (created with git-format) """
```

Comment:
I'd like to see a better summary as this says the same as the method name.
From what I see it creates a list of patch files metadata from given commits and patches.

Reply:
dude, you're so demanding; can do though